src/core/components/operation.jsx

```diff
@@ -107,7 +107,8 @@ export default class Operation extends PureComponent {
     const Schemes = getComponent( "schemes" )
     const OperationServers = getComponent( "OperationServers" )
     const OperationExt = getComponent( "OperationExt" )
-    const DeepLink = getComponent( "DeepLink" )
+    const DeepLink = getComponent("DeepLink")
```

Comment:
nit: add spaces around parameters to keep the file consistent?

Reply:
Good catch